src/repositories/ColorPaletteRepository.ts

```diff
@@ -76,7 +76,7 @@ export function mapFrontifyColorPaletteCreateToColorPaletteApiCreate(
 }
 
 export function mapFrontifyColorPalettePatchToColorPaletteApiPatch(
```

Comment:
We should update method name as well:
```suggestion
export function mapColorPalettePatchToColorPaletteApiPatch(
```